drivers/bridge/bridge.go

```diff
@@ -330,7 +330,7 @@ func (c *networkConfiguration) conflictsWithNetworks(id string, others []*bridge
 		// bridges. This could not be completely caught by the config conflict
 		// check, because networks which config does not specify the AddressIPv4
 		// get their address and subnet selected by the driver (see electBridgeIPv4())
```

Comment:
Even though this comment is a bit outdated (`electBridgeIPv4()` no longer exists), it is still true that the driver may assign the subnet for a network, it does so only for the default bridge and user specified linux bridge network.
(It happens also when reading from the store during initialization, but that one is safe.)

I am not sure yet why this is happening only now, but I am ok with the fix to avoid the panic.

Now this check is only a best effort, but we can investigate improvments later, because it would fail only in a very narrow case.